src/Data/converters/UserConverter.cs

```diff
@@ -0,0 +1,26 @@
+using Data.Entities;
+using Domain.utils;
+using Dto;
+using Dto.tools;
+
+namespace Data.converters;
+
+public static class UserConverter {
+    public static UserDto? ToEntity(AdminEntity? adminEntity) {
```

Comment:
should we change ToEntity to different name since we have 2 ToEntity

Reply:
Yep. That was an oversight on my part. That should have been ToDto . Will fix it.